data/test_data/gpx/export_test.gpx

```diff
@@ -6,7 +6,11 @@
 </metadata>
 <wpt lat="48.209846" lon="16.376023">
   <name>Point 1</name>
-  <cmt>Point 1</cmt>
+  <desc>Point 1</desc>
+</wpt>
+<wpt lat="48.209847" lon="16.376028">
+  <name><![CDATA[Point cdata name ><&"]]></name>
```

Comment:
Is there a test without the `<name>` tag?

Reply:
Yes